docs/reference/alert-schema.asciidoc

```diff
@@ -168,4 +173,11 @@ Type: long
 NOTE: This field only appears if you've set the {kibana-ref}/settings.html#server-publicBaseUrl[`server.publicBaseUrl`] configuration setting in the `kibana.yml` file. 
 
 Type: long
+
+|N/A | `kibana.alert.workflow_tags` a| List of tags added to an alert. 
+
+This field can contain an array of values, for example: `["production", "env2"]`
```

Comment:
Yeah, it could be a bit more realistic to use one of the default options here. I initially landed on these example values because users can create custom tags that describe anything, really. I'll update to include one of the default tags though:

```suggestion
This field can contain an array of values, for example: `["False Positive", "production"]`
```

Reply:
A combination of default and custom values sounds good!